### ISSUE TITLE
An important styling not included in css file

### ISSUE THREAD

**Issue by sukrosono** [danielm/uploader#109]
I was expecting the necessary styling as founded on the demo page are included in the /dist/css file.
It's turn out that it was [separated](https://github.com/danielm/uploader/blob/master/demo/index.html#L14), please let me know the reason.

At this time and for my current project. the rest css file are not used or at least not necessary as i use boostrap framework.
So the main styling i looking for is the border as it have functionality on drag and drop zone.

In conclusion, can we include the styling `style.css` to be included in dist file next release?
